### PR TITLE
Fix video tooltip

### PR DESCRIPTION
**Pull request by benjaminpkane**
## What changes are proposed in this pull request?

Adds the tooltip back to the modal video looker. Also fixes changing the color scheme for video lookers. See recording for current issue

https://github.com/user-attachments/assets/21daa6c0-6c1b-474e-8f7d-702fad93b507

### What areas of FiftyOne does this PR affect?

-   [x] App: FiftyOne application changes
-   [ ] Build: Build and test infrastructure changes
-   [ ] Core: Core `fiftyone` Python library changes
-   [ ] Documentation: FiftyOne documentation changes
-   [ ] Other

## Summary by CodeRabbit

- **New Features**
  - Enhanced state management for the modal looker, improving integration with the modal context.
  - Improved video frame management and playback functionality.
  - Added support for tracking multiple buffer ranges in video playback.

- **Bug Fixes**
  - Simplified the `ModalLookerNoTimeline` component, removing unused imports and unnecessary state management.
  - Streamlined state management in the `ColorFooter` component.

- **Documentation**
  - Updated import statements for clarity and consistency.

### CODE REVIEW

**Review by minhtuev — DISMISSED** [review]
LGTM!

**Review by findtopher — APPROVED** [review]
😋

**Review by findtopher — APPROVED** [review]
Now with more lint!

File: app/packages/core/src/components/ColorModal/ColorFooter.tsx [hunk 1]
```diff
@@ -26,8 +25,8 @@ const ColorFooter: React.FC = () => {
   const subscription = useRecoilValue(fos.stateSubscription);
 
   useEffect(
-    () => foq.subscribe(() => setActiveColorModalField(null)),
-    [setActiveColorModalField]
+    () => foq.subscribe((_, { set }) => set(activeColorEntry, null)),
+    []
   );
```
Comment (lanzhenw) [inline app/packages/core/src/components/ColorModal/ColorFooter.tsx:30]: nice

**Review by lanzhenw — APPROVED** [review]
Screenshot 2024-10-11 at 9 12 13 AM

Nice clean ups. color by value mode in the frame still have a minor issue.

**Comment by lanzhenw** [comment]
Screenshot 2024-10-11 at 9 28 46 AM

It's working. I selected the wrong field 😅

**Pull request merged**